src/SFA.DAS.AssessorService.Application/Handlers/Staff/StaffSearchHandler.cs

```diff
@@ -96,9 +96,9 @@ private async Task<StaffReposSearchResult> Search(StaffSearchRequest request)
                 // Search string is a long of 10 length so must be a uln.
                 var sr = new StaffReposSearchResult
```

Comment:
This code works, albeit is a bit smelly. Just wondering if this looks cleaner:

var results = await _staffIlrRepository.SearchForLearnerByUln(request);

return new StaffReposSearchResult
{
	PageOfResults = results,
	TotalCount = results.Count()
};